Register screen(peter)

plancer0 opened the pull request.

plancer0 committed:
Integracion de fuentes y temas

plancer0 committed:
Button Component

unhumanosinmas committed:
Componente TextInput

creacion de componente TextInput Personalizado

unhumanosinmas committed:
Merge remote-tracking branch 'origin/peter' into diego

unhumanosinmas committed:
Login Preliminar

Login de Referencia

plancer0 committed:
Inputs

plancer0 committed:
SVG Icons and register form

plancer0 committed:
Home page

unhumanosinmas committed:
Merge branch 'register_screen(peter)' into login_screen(diego)

unhumanosinmas committed:
3 views complete

se realizo adaptó la view del Login

unhumanosinmas committed:
Fixed Views

Color de los placeholders módificados

GibranPolonsky commented on src/components/buttons/buttons.style.tsx:
```diff
@@ -0,0 +1,21 @@
+  },
+  text: {
+    textAlign: 'center',
+    fontSize: 25,
```
Podemos mover los fontsize al archivo global de fuentes asi como hiciste con colores

GibranPolonsky commented on src/components/theme/fonts.ts:
```diff
@@ -0,0 +1,4 @@
+export const TITLES = 60;
+export const SUBTITLES = 30;
+export const TEXTO_NORMAL = 20;
```
En ingles en la medida de lo posible, REGULAR_TEXT, TINY_TEXT

GibranPolonsky commented on src/views/Home/home.styles.ts:
```diff
@@ -0,0 +1,21 @@
+    alignContent: 'center'
+  },
+  textPhrase: {textAlign: 'center', fontFamily: 'Poppins-Regular',},
+  titleHome: {fontSize: 45, fontFamily: 'Poppins-Black', textAlign: 'center'},
```
fontSize, dentro del archivo de fonts

GibranPolonsky commented on src/views/login/login.styles.tsx:
```diff
@@ -0,0 +1,62 @@
+    textAlign: 'center',
+  },
+  containerRegister: {alignItems: 'center', marginTop: 100, flex: 1},
+  inputStyle: {
```
Esto creo que lo podes abstraer y escribir el estilo de los inputs todo dentro del textinput component

GibranPolonsky commented on src/views/login/login.styles.tsx:
```diff
@@ -0,0 +1,62 @@
+    textAlignVertical: 'center',
+  },
+  buttonContainer: {paddingTop: 20, marginBottom: 60},
+  buttonsSignUpContainer: {
```
Igual aqui con botones, los componentes tienen que tener estos estilos, las vistas no

GibranPolonsky commented on package.json:
```diff
@@ -11,7 +11,9 @@
-    "react-native": "0.68.2"
+    "react-native": "0.68.2",
+    "react-native-svg": "12.3.0",
+    "react-native-svg-transformer": "1.0.0"
```
para que usamos esta libreria?

plancer0 replied:
Para poder usar archivos SVG como iconos

plancer0 committed:
Correcion de primera pull request

unhumanosinmas committed:
FIxed Login InputText

se arreglaron los imputs

unhumanosinmas committed:
Fixed InputText Register

Se hizo uso del componente customizado del InputText

plancer0 committed:
Identación de código

Se idento el codigo y se empezo a implementar la navegacion

plancer0 committed:
Merge remote-tracking branch 'origin/login_screen(diego)' into register_screen(peter)

plancer0 committed:
Arreglo de rutas

Movi la carpeta utils a src y actualice rutas

plancer0 committed:
Arreglo de redundancias y errores

Saque codigo de estilo que no servia o que provocaban redundancias y arregle un error en los componentes, arregle mucha parte de los componentes boton e input

plancer0 committed:
Navigation

Se implemento la navegacion el proyecto

plancer0 committed:
Iconos y estilo
